cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go

```diff
@@ -116,6 +116,8 @@ type ClusterConfiguration struct {
 // ControlPlaneComponent holds settings common to control plane component of the cluster
 type ControlPlaneComponent struct {
 	// ExtraArgs is an extra set of flags to pass to the control plane component.
+	// The keys in this map are the flag name as it appears on the
```

Comment:
Plural vs singular:
Should this be 'A key in this map is a flag...'

Reply:
Thanks, fixed.